src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx

```diff
@@ -81,24 +81,42 @@ function UserStats({totals, location, organization, transactionName}: Props) {
 
   return (
     <Container>
-      <div>
-        <SectionHeading>{t('Apdex Score')}</SectionHeading>
-        <StatNumber>{apdex}</StatNumber>
-        <Link to={`/settings/${organization.slug}/performance/`}>
-          <SectionValue>
-            {threshold}ms {t('threshold')}
-          </SectionValue>
-        </Link>
-      </div>
       {vitalsPassRate !== null && (
         <div>
-          <SectionHeading>{t('Web Vitals')}</SectionHeading>
+          <SectionHeading>
+            {t('Web Vitals')}
+            <QuestionTooltip
+              position="top"
+              title={t(
+                'Based on Google Web Vitals, this is the number of passing web vitals that meet the requirements for "good" or "needs improvement".'
+              )}
+              size="sm"
+            />
+          </SectionHeading>
           <StatNumber>{vitalsPassRate}</StatNumber>
           <Link to={webVitalsTarget}>
             <SectionValue>{t('Passed')}</SectionValue>
           </Link>
         </div>
       )}
+      <div>
```

Comment:
I believe putting apdex second can result in web vitals disappearing as it loads and no web vital data is found.